op-signer/client/client.go

```diff
@@ -25,35 +25,41 @@ type SignerClient struct {
 }
 
 func NewSignerClient(logger log.Logger, endpoint string, tlsConfig optls.CLIConfig) (*SignerClient, error) {
-	caCert, err := os.ReadFile(tlsConfig.TLSCaCert)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read tls.ca: %w", err)
-	}
-	caCertPool := x509.NewCertPool()
-	caCertPool.AppendCertsFromPEM(caCert)
+	var httpClient *http.Client
```

Comment:
Could you add a `log.Info` for both cases, just so we know at runtime if tls is being used or not?